flake-modules/dev/list-plugins/default.nix

```diff
@@ -19,6 +21,17 @@
         ];
       } (builtins.readFile ./list-plugins.py);
 
+      checks.list-plugins-test =
+        pkgs.runCommand "list-plugins-test"
+          {
+            nativeBuildInputs = [
+              self'.packages.list-plugins
+            ];
+          }
+          ''
+            list-plugins --root-path ${toString self} > $out
+          '';
```

Comment:
This feels more like a test _of_ the package itself rather than a general test of our repo?

Therefore I'd _prefer_ to see this defined locally to the package, i.e. as `list-plugins.passthru.test`

We could still add it to the flake checks; in the same way we do for the `docs.user-configs.tests` tests.

Reply:
yes, I wanted to do this too, but:
1) I need to pass the absolute path to the flake (`self`)
2) I don't think that I can add a test or `passthru` with `writePython3Bin`.

Reply:
> 1. I need to pass the absolute path to the flake (`self`)

Hm, I'm not sure why _that_ is a blocker?

> * I don't think that I can add a test or `passthru` with `writePython3Bin`.

Hm, maybe that's not made accessible? But it should be possible via `.overrideAttrs` in the worst case.